examples/docker/modin-hdk/utils.py

```diff
@@ -0,0 +1,28 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import sys
+import time
+from os.path import dirname, realpath
+
+MODIN_DIR = dirname(dirname(dirname(dirname(realpath(__file__)))))
```

Comment:
Since we get MODIN_DIR in several places using the same way I suggest doing like this.
https://github.com/modin-project/modin/blob/0215a133ac32969ca0a4126f6aef82be099ae8ef/examples/tutorial/jupyter/execution/pandas_on_ray/test/test_notebooks.py#L19